geth/txqueue/txqueue_manager.go

```diff
@@ -190,6 +190,8 @@ func (m *Manager) CompleteTransaction(id common.QueuedTxID, password string) (ge
 	return hash, err
 }
 
+const timeoutTime = time.Minute
```

Comment:
`timeoutTime` doesn't say anything about the meaning of the variable, it can be any timeout. Mind renaming it to something at least like `ctxCancelTimeout` or adding a comment or both?

Reply:
Ok.